entrypoint.sh

```diff
@@ -19,7 +19,7 @@ urlencode() {
 
 # This supports both the old environment variables and new ones specific to the Figma setup.
 if [ -n "$PG_FIGMA_FOR_JIRA_DB_ROLE" ]; then
-  DATABASE_URL="postgresql://${PG_FIGMA_FOR_JIRA_DB_ROLE}:${PG_FIGMA_FOR_JIRA_DB_PASSWORD}@${PG_FIGMA_FOR_JIRA_DB_HOST}:${PG_FIGMA_FOR_JIRA_DB_PORT}/${PG_FIGMA_FOR_JIRA_DB_SCHEMA}"
+  DATABASE_URL="postgresql://${PG_FIGMA_FOR_JIRA_DB_ROLE}:${PG_FIGMA_FOR_JIRA_DB_PASSWORD}@${PG_FIGMA_FOR_JIRA_DB_HOST}:${PG_FIGMA_FOR_JIRA_DB_PORT}/${PG_FIGMA_FOR_JIRA_DB_SCHEMA}?sslmode=${PG_FIGMA_FOR_JIRA_DB_SSL_MODE}"
 else
   ENCODED_PASSWORD="$(urlencode "$PG_INTEGRATIONS_JIRA_PASSWORD")"
   DATABASE_URL="postgresql://${PG_INTEGRATIONS_JIRA_USER}:${ENCODED_PASSWORD}@${PG_INTEGRATIONS_HOST}:${PG_INTEGRATIONS_PORT}/${PG_INTEGRATIONS_DATABASE}"
```

Comment:
**question:** @Roystbeef , @elainefigma , if I understand correctly, the app deployed on Figma's infrastructure uses this `DATABASE_URL` value. If this is correct, the change should have no effect for Figma's deployment.